Packager/Factories/CodingHistory/DatCodingHistoryGenerator.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Packager.Extensions;
+using Packager.Models.FileModels;
+using Packager.Models.PodMetadataModels;
+using System.Collections;
```

Comment:
VS is telling me that `using System.Collections` can be removed.

Reply:
Resolved https://github.com/IUMDPI/IUMediaHelperApps/pull/29/commits/9257104f1a655b089a7c7d44db46738ccada92f9